### DISCUSSION TITLE
borg backup - force killed via container stop during backup - now database is not starting saying waiting for backup to finish

### DISCUSSION THREAD

**Discussion by fate8383** [nextcloud/all-in-one#1469] [Questions] [answered]
basically I didn't know data folder is not excluded from backup and thus having a huge data folder I cannot use Borg, so when almost reached my limit in the backup storage I had to stop the Borg container to stop backup. started all compose and now database container does not start saying "waiting for back up to finish. I tried all I can as in remove database-dumb volume to remove etc... not allowing me as it is in use. I am suing portioner btw. What can I do to get this back, I already disabled backup on config json. thanks

**Comment by fate8383** [comment]
couple more tries and now nextcloud-aio-database container vanished. leaving volumes and image unused. What can I do now?

**Comment by fate8383** [comment]
Slim Application Error
The application could not run because of the following error:

Details

Type: GuzzleHttp\Exception\ClientException
Code: 404
Message: Client error: `GET http://localhost/v1.41/containers/nextcloud-aio-database/logs?stdout=true&stderr=true` resulted in a `404 Not Found` response: {"message":"No such container: nextcloud-aio-database"}
File: /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php
Line: 113

**Comment by szaimen** [comment, marked as answer]
`sudo rm /var/lib/docker/volumes/nextcloud_aio_database_dump/_data/backup-is-running` should have fixed the initial problem.

**Comment by szaimen** [comment]
As for the other problem, try to reload the AIO interface and stop & start the containers again.

**Reply by fate8383** [reply]
I don't have the database container to start stop, how do I get it back? I mean it is visible in the AIO UI but give the errror I pasted, I am trying from portioner side to start top but I don't have the container there.

**Reply by szaimen** [reply]
as I said, please reload the AIO interface and try to stop and start the container again

**Reply by fate8383** [reply]
Containers

 Apache ([Stopped](https://192.168.23.136:8080/api/docker/logs?id=nextcloud-aio-apache))
 Database ([Stopped](https://192.168.23.136:8080/api/docker/logs?id=nextcloud-aio-database))
 Nextcloud ([Stopped](https://192.168.23.136:8080/api/docker/logs?id=nextcloud-aio-nextcloud))
 Redis ([Stopped](https://192.168.23.136:8080/api/docker/logs?id=nextcloud-aio-redis))
⚠️ Container updates are available. Click on 'Stop Containers' and 'Start Containers' to update them. You should consider creating a backup first.

this is what I see in the interface, reloaded, restarted stack, individual container of the master AIO and this is the view I get above. So from there If say start and update I am getting errors I pasted already.

**Reply by szaimen** [reply]
Can you post the whole `nextcloud-aio-mastercontainer` logs here?

**Reply by fate8383** [reply]
ok full docker service restart fixed it but I have a new problem now :) when I go to my nextcloud instance:
Internal Server Error

The server encountered an internal error and was unable to complete your request.
Please contact the server administrator if this error reappears multiple times, please include the technical details below in your report.
More details can be found in the server log.

**Reply by szaimen** [reply]
> Internal Server Error
> 
> The server encountered an internal error and was unable to complete your request. Please contact the server administrator if this error reappears multiple times, please include the technical details below in your report. More details can be found in the server log.

Are all containers running correctly now? When you try to restart the containers from the AIO interface, does it work now?

**Reply by szaimen** [reply]
> Internal Server Error
> The server encountered an internal error and was unable to complete your request. Please contact the server administrator if this error reappears multiple times, please include the technical details below in your report. More details can be found in the server log.

Are all containers running correctly now? When you try to restart the containers from the AIO interface, does it work now?

**Reply by fate8383** [reply]
they say they are all working but I see tens of these in the database container logs:
2022-12-01 16:31:46.404 CET [115] ERROR:  relation "oc_appconfig" does not exist at character 15
2022-12-01 16:31:46.404 CET [115] STATEMENT:  SELECT * FROM "oc_appconfig"

> 

and I cannot access next cloud as the internal server error says above

**Reply by szaimen** [reply]
> they say they are all working but I see tens of these in the database container logs: 2022-12-01 16:31:46.404 CET [115] ERROR: relation "oc_appconfig" does not exist at character 15 2022-12-01 16:31:46.404 CET [115] STATEMENT: SELECT * FROM "oc_appconfig"
> 
> > 
> 
> and I cannot access next cloud as the internal server error says above

great! Seems like you delete too much then.

Please either restore a backup from before the whole disaster or start from scratch: https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance

**Reply by fate8383** [reply]
and tens of these in the nextcloud container logs:
Next Doctrine\DBAL\Exception\TableNotFoundException: An exception occurred while executing a query: SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "oc_appconfig" does not exist
LINE 1: SELECT * FROM "oc_appconfig"
                      ^ in /var/www/html/3rdparty/doctrine/dbal/src/Driver/API/PostgreSQL/ExceptionConverter.php:67

ace:
#0 /var/www/html/3rdparty/doctrine/dbal/src/Connection.php(1780): Doctrine\DBAL\Driver\API\PostgreSQL\ExceptionConverter->convert(Object(Doctrine\DBAL\Driver\PDO\Exception), Object(Doctrine\DBAL\Query))
#1 /var/www/html/3rdparty/doctrine/dbal/src/Connection.php(1719): Doctrine\DBAL\Connection->handleDriverException(Object(Doctrine\DBAL\Driver\PDO\Exception), Object(Doctrine\DBAL\Query))
#2 /var/www/html/3rdparty/doctrine/dbal/src/Connection.php(1067): Doctrine\DBAL\Connection->convertExceptionDuringQuery(Object(Doctrine\DBAL\Driver\PDO\Exception), 'SELECT * FROM "...', Array, Array)
#3 /var/www/html/lib/private/DB/Connection.php(261): Doctrine\DBAL\Connection->executeQuery('SELECT * FROM "...', Array, Array, NULL)
#4 /var/www/html/3rdparty/doctrine/dbal/src/Query/QueryBuilder.php(345): OC\DB\Connection->executeQuery('SELECT * FROM "...', Array, Array)
#5 /var/www/html/lib/private/DB/QueryBuilder/QueryBuilder.php(281): Doctrine\DBAL\Query\QueryBuilder->execute()
#6 /var/www/html/lib/private/AppConfig.php(411): OC\DB\QueryBuilder\QueryBuilder->execute()
#7 /var/www/html/lib/private/AppConfig.php(176): OC\AppConfig->loadConfigValues()
#8 /var/www/html/lib/private/AppConfig.php(367): OC\AppConfig->getApps()
#9 /var/www/html/lib/private/legacy/OC_App.php(983): OC\AppConfig->getValues(false, 'installed_versi...')
#10 /var/www/html/lib/private/Server.php(713): OC_App::getAppVersions()
#11 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(162): OC\Server->OC\{closure}(Object(OC\Server))
#12 /var/www/html/3rdparty/pimple/pimple/src/Pimple/Container.php(122): OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}(Object(Pimple\Container))
#13 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(129): Pimple\Container->offsetGet('OC\\Memcache\\Fac...')
#14 /var/www/html/lib/private/ServerContainer.php(136): OC\AppFramework\Utility\SimpleContainer->query('OC\\Memcache\\Fac...', false)
#15 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(57): OC\ServerContainer->query('OC\\Memcache\\Fac...')
#16 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(184): OC\AppFramework\Utility\SimpleContainer->get('OC\\Memcache\\Fac...')
#17 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(162): OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}(Object(OC\Server))
#18 /var/www/html/3rdparty/pimple/pimple/src/Pimple/Container.php(118): OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}(Object(Pimple\Container))
#19 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(129): Pimple\Container->offsetGet('OCP\\ICacheFacto...')
#20 /var/www/html/lib/private/ServerContainer.php(136): OC\AppFramework\Utility\SimpleContainer->query('OCP\\ICacheFacto...', false)
#21 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(57): OC\ServerContainer->query('OCP\\ICacheFacto...')
#22 /var/www/html/lib/private/Server.php(1106): OC\AppFramework\Utility\SimpleContainer->get('OCP\\ICacheFacto...')
#23 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(162): OC\Server->OC\{closure}(Object(OC\Server))
#24 /var/www/html/3rdparty/pimple/pimple/src/Pimple/Container.php(122): OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}(Object(Pimple\Container))
#25 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(129): Pimple\Container->offsetGet('OCP\\Lock\\ILocki...')
#26 /var/www/html/lib/private/ServerContainer.php(136): OC\AppFramework\Utility\SimpleContainer->query('OCP\\Lock\\ILocki...', false)
#27 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(57): OC\ServerContainer->query('OCP\\Lock\\ILocki...')
#28 /var/www/html/lib/private/Server.php(2083): OC\AppFramework\Utility\SimpleContainer->get('OCP\\Lock\\ILocki...')
#29 /var/www/html/lib/private/Files/View.php(122): OC\Server->getLockingProvider()
#30 /var/www/html/lib/private/Server.php(454): OC\Files\View->__construct()
#31 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(162): OC\Server->OC\{closure}(Object(OC\Server))
#32 /var/www/html/3rdparty/pimple/pimple/src/Pimple/Container.php(122): OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}(Object(Pimple\Container))
#33 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(129): Pimple\Container->offsetGet('OC\\Files\\Node\\H...')
#34 /var/www/html/lib/private/ServerContainer.php(136): OC\AppFramework\Utility\SimpleContainer->query('OC\\Files\\Node\\H...', false)
#35 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(57): OC\ServerContainer->query('OC\\Files\\Node\\H...')
#36 /var/www/html/lib/private/Server.php(1446): OC\AppFramework\Utility\SimpleContainer->get('OC\\Files\\Node\\H...')
#37 /var/www/html/lib/base.php(602): OC\Server->boot()
#38 /var/www/html/lib/base.php(1111): OC::init()
#39 /var/www/html/cron.php(43): require_once('/var/www/html/l...')
#40 {main}
[2022-12-01 16:43:56.428175 +01:00] WARN [notify_push::connection] src/connection.rs:75: Error while sending authentication request to nextcloud
[2022-12-01 16:45:56.424925 +01:00] WARN [notify_push::connection] src/connection.rs:75: Error while sending authentication request to nextcloud

**Reply by szaimen** [reply]
> they say they are all working but I see tens of these in the database container logs: 2022-12-01 16:31:46.404 CET [115] ERROR: relation "oc_appconfig" does not exist at character 15 2022-12-01 16:31:46.404 CET [115] STATEMENT: SELECT * FROM "oc_appconfig"
> 
> > 
> 
> and I cannot access next cloud as the internal server error says above

great! Seems like you delete too much then.

Please either restore a backup from before the whole disaster or start from scratch: https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance

**Reply by fate8383** [reply]
> > they say they are all working but I see tens of these in the database container logs: 2022-12-01 16:31:46.404 CET [115] ERROR: relation "oc_appconfig" does not exist at character 15 2022-12-01 16:31:46.404 CET [115] STATEMENT: SELECT * FROM "oc_appconfig"
> > > 
> > 
> > 
> > and I cannot access next cloud as the internal server error says above
> 
> great! Seems like you delete too much then.
> 
> Please either restore a backup from before the whole disaster or start from scratch: https://github.com/nextcloud/all-in-one#how-to-properly-reset-the-instance

I was almost convinced to do that, I guess that dump folder rm -rf was a little too much... :( thanks

**Reply by szaimen** [reply]
> I was almost convinced to do that, I guess that dump folder rm -rf was a little too much... :( thanks

yes indeed

**Comment by fate8383** [comment]
I have done full rm -rf to _dump folder, restarted all now and still not starting
Trace

#0 /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Middleware.php(69): GuzzleHttp\Exception\RequestException::create(Object(GuzzleHttp\Psr7\Request), Object(GuzzleHttp\Psr7\Response), NULL, Array, NULL)
#1 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(204): GuzzleHttp\Middleware::GuzzleHttp\{closure}(Object(GuzzleHttp\Psr7\Response))
#2 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(153): GuzzleHttp\Promise\Promise::callHandler(1, Object(GuzzleHttp\Psr7\Response), NULL)
#3 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/TaskQueue.php(48): GuzzleHttp\Promise\Promise::GuzzleHttp\Promise\{closure}()
#4 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(248): GuzzleHttp\Promise\TaskQueue->run(true)
#5 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(224): GuzzleHttp\Promise\Promise->invokeWaitFn()
#6 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(269): GuzzleHttp\Promise\Promise->waitIfPending()
#7 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(226): GuzzleHttp\Promise\Promise->invokeWaitList()
#8 /var/www/docker-aio/php/vendor/guzzlehttp/promises/src/Promise.php(62): GuzzleHttp\Promise\Promise->waitIfPending()
#9 /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Client.php(187): GuzzleHttp\Promise\Promise->wait()
#10 /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/ClientTrait.php(44): GuzzleHttp\Client->request('GET', 'http://localhos...', Array)
#11 /var/www/docker-aio/php/src/Docker/DockerActionManager.php(160): GuzzleHttp\Client->get('http://localhos...')
#12 /var/www/docker-aio/php/src/Controller/DockerController.php(56): AIO\Docker\DockerActionManager->GetLogs('nextcloud-aio-d...')
#13 /var/www/docker-aio/php/vendor/slim/slim/Slim/Handlers/Strategies/RequestResponse.php(38): AIO\Controller\DockerController->GetLogs(Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response), Array)
#14 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(358): Slim\Handlers\Strategies\RequestResponse->__invoke(Array, Object(GuzzleHttp\Psr7\ServerRequest), Object(GuzzleHttp\Psr7\Response), Array)
#15 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\Routing\Route->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#16 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#17 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/Route.php(315): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#18 /var/www/docker-aio/php/vendor/slim/slim/Slim/Routing/RouteRunner.php(68): Slim\Routing\Route->run(Object(GuzzleHttp\Psr7\ServerRequest))
#19 /var/www/docker-aio/php/vendor/slim/csrf/src/Guard.php(476): Slim\Routing\RouteRunner->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#20 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(168): Slim\Csrf\Guard->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Slim\Routing\RouteRunner))
#21 /var/www/docker-aio/php/vendor/slim/twig-view/src/TwigMiddleware.php(115): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#22 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Views\TwigMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#23 /var/www/docker-aio/php/src/Middleware/AuthMiddleware.php(38): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#24 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(269): AIO\Middleware\AuthMiddleware->__invoke(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#25 /var/www/docker-aio/php/vendor/slim/slim/Slim/Middleware/ErrorMiddleware.php(76): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#26 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(121): Slim\Middleware\ErrorMiddleware->process(Object(GuzzleHttp\Psr7\ServerRequest), Object(Psr\Http\Server\RequestHandlerInterface@anonymous))
#27 /var/www/docker-aio/php/vendor/slim/slim/Slim/MiddlewareDispatcher.php(65): Psr\Http\Server\RequestHandlerInterface@anonymous->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#28 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(199): Slim\MiddlewareDispatcher->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#29 /var/www/docker-aio/php/vendor/slim/slim/Slim/App.php(183): Slim\App->handle(Object(GuzzleHttp\Psr7\ServerRequest))
#30 /var/www/docker-aio/php/public/index.php(167): Slim\App->run()
#31 {main}
